### PR TITLE
Add loan history to the carrier's show page

### LINKED ISSUE #282: Add a Loan history on the Carrier Show page
We would like to have a history of the loans on a carrier so I we can know where the carrier has been.

We have a similar Loan History on the User, /user/:id page, we should try to make the implementation and look similar to this table.

Babywearing

### PR DESCRIPTION
**Pull request by pollygee**
Resolves #282 

Added the Loan history of a carrier to it's show page.

### CODE REVIEW

File: spec/features/carrier_spec.rb [hunk 1]
```diff
@@ -51,6 +52,7 @@
       expect(page).to have_content('test carrier')
       expect(page).to have_content('babywearing')
       expect(page).to have_content('test model')
+      expect(page).to have_content(loan.checkout_volunteer)
```
Comment (pollygee) [inline spec/features/carrier_spec.rb:55]: I could add all the columns here.  Didn't know if that was really needed.  wdyt?

Comment (mark100net) [reply spec/features/carrier_spec.rb:55]: Just a display thing. Not a big deal unless it drops coverage.

File: app/controllers/carriers_controller.rb [hunk 2]
```diff
@@ -39,6 +39,10 @@ def create
     end
   end
 
+  def show
+    @loans = Loan.where(carrier: @carrier)
```
Comment (mark100net) [inline app/controllers/carriers_controller.rb]: @carrier.loans is better

Comment (mark100net) [inline app/controllers/carriers_controller.rb:43]: `@loans = @carrier.loans` is better

**Review by mark100net — CHANGES_REQUESTED** [review]

**Review by mark100net — APPROVED** [review]

**Pull request merged**